lib/application/configure_new_cbj_comp/configure_new_cbj_comp_bloc.dart

```diff
@@ -294,7 +295,7 @@ class ConfigureNewCbjCompBloc
     final List<GenericLightDE> deviceEntityList = [];
 
     final String roomUuid = const Uuid().v1();
-    final String roomName = _textEditingController['allInSameRoom']!.text;
+    final String roomName = _textEditingController['allInSameRoom'.tr()]!.text;
```

Comment:
As this text is to get elements from an array maybe you meant to do `_textEditingController['allInSameRoom']!.text.tr()`

But I think it is better to remove this change entirely because it will request some testing to see that nothing breaks.